CHANGELOG.rst

```diff
@@ -1,6 +1,13 @@
 Changelog for Mantarray Desktop App
 ===================================
 
+1.0.7 (2022-12-16)
+
+Changed:
+^^^^^^^^
+- Order of metrics sent for heat map
```

Comment:
This should say something about how Twtich Freq is now the default heatmap metric, and that we change the default max/min